sensor_msgs/PointCloud2 cloud
---
float64[] feature_vector
